doc/releases/squid.rst

```diff
@@ -0,0 +1,335 @@
+  on prior releases, use these new tools to check whether their buckets are
+  affected and to clean them up accordingly.
+* RGW: The User Accounts feature unlocks several new AWS-compatible IAM APIs
+  for the self-service management of users, keys, groups, roles, policy and
+  more. Existing users can be adopted into new accounts. This process is
+  optional but irreversible. See https://docs.ceph.com/en/squid/radosgw/account
+  and https://docs.ceph.com/en/squid/radosgw/iam for details.
+* RGW: On startup, radosgw and radosgw-admin now validate the ``rgw_realm``
+  config option. Previously, they would ignore invalid or missing realms and go
+  on to load a zone/zonegroup in a different realm. If startup fails with a
+  "failed to load realm" error, fix or remove the ``rgw_realm`` option.
+* RGW: The radosgw-admin commands ``realm create`` and ``realm pull`` no longer
+  set the default realm without ``--default``.
+* RGW: Fixed an S3 Object Lock bug with PutObjectRetention requests that
+  specify a RetainUntilDate after the year 2106. This date was truncated to 32
+  bits when stored, so a much earlier date was used for object lock
+  enforcement.  This does not effect PutBucketObjectLockConfiguration where a
+  duration is given in Days.  The RetainUntilDate encoding is fixed for new
+  PutObjectRetention requests, but cannot repair the dates of existing object
+  locks. Such objects can be identified with a HeadObject request based on the
+  x-amz-object-lock-retain-until-date response header.
+* S3 ``Get/HeadObject`` now supports the query parameter ``partNumber`` to read
+  a specific part of a completed multipart upload.
+* RGW: The SNS CreateTopic API now enforces the same topic naming requirements
+  as AWS: Topic names must be made up of only uppercase and lowercase ASCII
+  letters, numbers, underscores, and hyphens, and must be between 1 and 256
+  characters long.
+* RGW: Notification topics are now owned by the user that created them.  By
+  default, only the owner can read/write their topics. Topic policy documents
+  are now supported to grant these permissions to other users. Preexisting
+  topics are treated as if they have no owner, and any user can read/write them
+  using the SNS API.  If such a topic is recreated with CreateTopic, the
+  issuing user becomes the new owner.  For backward compatibility, all users
+  still have permission to publish bucket notifications to topics owned by
+  other users. A new configuration parameter,
+  ``rgw_topic_require_publish_policy``, can be enabled to deny ``sns:Publish``
+  permissions unless explicitly granted by topic policy.
+* RGW: Fix issue with persistent notifications where the changes to topic param
+  that were modified while persistent notifications were in the queue will be
+  reflected in notifications.  So if user sets up topic with incorrect config
+  (password/ssl) causing failure while delivering the notifications to broker,
```

Comment:
Something is strange in the sentence starting with 'So if user...'

Reply:
Fixed